virttest/tests/unattended_install.py

```diff
@@ -168,6 +168,10 @@ def __init__(self, test, params, vm):
         for a in self.attributes:
             setattr(self, a, params.get(a, ''))
 
+        # Make finish.bat work well with positional arguments
+        if not self.process_check.strip():  # pylint: disable=E0203
+            self.process_check = '""'       # pylint: disable=E0203
```

Comment:
Could you explain the difference between`''` and `'""'`. The old value for `self.process_check` is `''` if it was not defined in params.
```
 168         for a in self.attributes:                                               
 169             setattr(self, a, params.get(a, '')) 
```

Reply:
I think i have got your point, before only one useful param 'PROCESS_CHECK' for finish.bat, so run with `finish.bat PROCESS_CHECK`,  then `[%2]==[]` is True,  `[%1]==[]` depends on the value of `process_check`.
Now, with the third params,  since `[""]==[]` is false,  so need  remove `"`, replace `%2 ` with `%~2`.  Is it right?

If so, LGTM.